### ISSUE TITLE
rectify EB <> Meetup divergence

### ISSUE THREAD

**Issue by lianilychee** [open-austin/org#392]

**Comment by lianilychee** [comment]
I discovered this today.  I've emailed and Slacked the fellow.

**Comment by lianilychee** [comment]
The fellow has removed all the events on Meetup which is good... and also removed the redirect to Eventbrite that I put in the Meetup description in September.  Did not receive any response from him when I initially reached out last month

**Comment by lianilychee** [comment]
Deprioritized for now, since Jan-March became very busy

**Comment by lianilychee** [comment]
Aslan called the fellow today, and the fellow refused to hand back the reins.  We will go the policy@meetup.com route

**Comment by lianilychee** [comment]
We have 2mo on the clock.  Need to figure out transition plan

**Comment by jcklpe** [comment]
If the work of maintaining the Meetup does not fall on Liani's shoulders, then the only issue here is money right? Which they do have a free starter plan. I think there's a lot of social capital here we shouldn't leave on the table. If we were to do a full transition I think it would be good to have some of that actually demonstrated in numbers. What's our stats for Eventbrite versus Meetup etc. And that might need more than 2 months to really get solid numbers on.

**Comment by hlupico** [comment]
Can I be removed from these emails?

**Comment by lianilychee** [comment]
@hlupico if you're talking about the Open Austin newsletter, please submit an [subscribe/unsubscribe request](https://docs.google.com/forms/d/1LO-wP1-R7IYnW8ZHanvQ1FK4LKNlq4iu2QFLxsD-20Q/edit).  If you're talking about GH notifications, here's a [GH doc on changing your personal settings](https://docs.github.com/en/account-and-profile/managing-subscriptions-and-notifications-on-github/setting-up-notifications/configuring-notifications)

**Comment by lianilychee** [comment]
@jcklpe i renamed the ticket title.  also, I put a draft transition plan in the ticket body.  What do you think?  Plis feel free to change it however u see fit.  I'm happy to divvy up any of the transition responsibilities too, just lmk

**Comment by hlupico** [comment]
Thank you

**Comment by jcklpe** [comment]
@lianilychee I've asked in the past that we make an actual case for the transition before making the decision either way and I don't think that case has been made. Your outline above assumes this as a forgone conclusion.

**Comment by lianilychee** [comment]
My bad, I'm going off the latest you wrote on the discord group chat, "let's just get some time owning meetup, and we'll figure out next steps later"

**Comment by jcklpe** [comment]
I've created a survey for us to send out over various channels to get people's preferences in terms of platform: https://docs.google.com/forms/d/1gRbrsHy1FrPc16M_sJpwUhQTn3uEl9HdvMkDAX-3kvs/preview

**Comment by jcklpe** [comment]
@lianilychee It looks like we can but no emails are attached. It looks maybe it can with a Pro account but not sure: https://www.perplexity.ai/search/meetup-download-member-contact-TTXex9ejQ5ysjPJjKpwj1A

**Comment by jcklpe** [comment]
@lianilychee  Could you share those photos you took of the event platform participation at the March CoP, perhaps to the google drive?

**Comment by lianilychee** [comment]
@jcklpe , sure thing, which directory do u recommend?
* Core Team \ Practices \ Design
* Core Team \ Events \ <create an event-specific folder>
* Core Team \ Visual Assets \ <probs create a Photos folder>

**Comment by jcklpe** [comment]
@lianilychee  I was talking about the specific photos related to people's feedback on the event platform strategy. I think that issue should go under Practices / Design. Photos that are general parts of our PR, helping build a backlog of photos of people at events and stuff, should probably go in a Photos folder as you suggested. 

I'm down for google photos. I haven't used it extensively myself.

**Comment by lianilychee** [comment]
event platform photos - done

google photos thru the admin@ account - done

**Comment by lianilychee** [comment]
hey @jcklpe , friendly reminder that our 6-mo Meetup trial is ending in ~2 months!  As I've gone through several applications +  convos and outreach with various potential parters for our programs, it's increasingly clear to me that 

* (A) We need to have a single event platform to effectively measure our reach, communicate with our attendees, and better utilize that data to continue building our community
* (B) It's a disproportionate burden on me to hunt down RSVP and check-in numbers across all three platforms
* (C) There are low-to-no network effects based on event platform.

Over the next two months I'd like to start transitioning Open Austin to a single event platform.  But I also wanted to message/speak with u first because I want to build this together.  Does this work for you?

**Comment by jcklpe** [comment]
@lianilychee  Thank you for circling back on this. Time has gone by fast. A few thoughts:
- I def don't want to burden you or anyone with additional work. 
- I don't know if we need to be hunting down or collecting RSVP and check-in numbers across 3 platforms? Do we have a doc where we are doing that? 
- I'm not convinced on point C. I'd like us to review the numbers explicitly. I'm not trying to hold out on this subject, I just want us to have an actual structured conversation around it. I'm going to go ahead and get a Meetup Pro free trial though so that I can begin grabbing all the member emails for us to make it possible to get off Meetup if it comes to that.

**Comment by lianilychee** [comment]
I changed the unordered bullets to numbered ones to make points eaasier for us to refer to!

(1) 👍 

(2) Having gone thru multiple partner convos, outreach efforts, and funder applications, we absolutely do need RSVP and check-in numbers:

* **Funding applications require overall attendance and unduplicated attendance over time.**  I've spoken with a few org leaders at more mature organizations and they all went thru the "let's test them all!" situation that we did, now they have ONE platform that has robust analytics and it's EB.  The odd org out uses a paid [Bonterra](https://www.bonterratech.com/) product that combines fundraising, event management, and newsletter.  
  * Example, that application that you wrote an answer to.  They required Open Austin to have reached at least 200 folks in the past 12 months.  I used pictures and EB to sum up the data practice meet attendees.  I honestly just guessed/averaged for the design practice ones, based on the ones I've been to.
* **We can be more strategic about what holes we have, because we have better visibility**.  The other org leaders all agree from their experience that going forward with one allowed them to grow capability in other areas.
  * Exaggerated example 1: Let's say we have 200 attendees from the past 12 months.
    * Are they 200 unique attendees?  What do they attend?  
    * Are they the same 10 people repeatedly over 20 events?  Maybe the same person has come to 20 practice meets.  Then we should really encourage them to fill in a contributor profile and thank them for being a loyal fixture of our community!  Maybe the person comes to some practice meets, some socials.  Then we should pick their brains about word-of-mouth and where we can do outreach.
  * Example 2: If we have 100 RSVPs for an event but only 5 check-ins, then that means we have to be better about pre-event engagement and offering incentives for showing up, like "bring a +1 and get a free raffle ticket"

"Is there a doc that we're tracking" - I'm pulled [this](https://docs.google.com/spreadsheets/d/1MSVLu1zARLuSzSsi1QwmnBIZjEZYiesqX4eUEu_h7kM/edit?gid=0#gid=0_ together for the application that you wrote an answer for

(3) Haha I don't think you're holding out and I love that you want to be thorough.  And at the same time, I want Open Austin to have the infrastructure that it actually needs and how it actually works, not copying legacy models (sorry to throw your ACT board member pitch back at you; I thought it was good), so that's why I'm bringing new data to the table.  And because the world doesn't run on infinite money and time, eventually we'll have to make decisions with imperfect data.  I think your "which platform do you prefer" survey is a good data point, here are the other pieces:
* My Point 2, that we absolutely do need those numbers.  Scattered numbers is an undue burden.
* I will confidently say that network effects have zero bearing on the data practice meet attendance
* For monthly volunteer orientation, if ppl aren't finding us thru EB, they're finding us thru social media, word of mouth, tabling, and "searching" (which is Google or ChatGPT).  

Happy to pair with u to sift thru whatever if helpful!  Though I don't have skills that you don't.  I think Meetup Pro is a good move.  LMK if you want me to plug and chug anything around that with you

**Comment by lianilychee** [comment]
closing this ticket, following aslan/liani dm in slack.  i'll be moving on to cutover plan which is also outlined in the copy/pasted convo below

---
Aslan French

> I think it's worth getting the email list out of the system at the very least.
> IMO the way to make this decision properly would be to actually have the Board vote on it etc. But we don't even really have a functional board at this point. And I'm not really willing to fight this issue. But I do want to close things out properly and that means 2 things:
> 1. getting the emails out of the system
> 2. closing down the account properly so it can't be used by anyone else.

Liani Lye [2:02 PM](https://open-austin.slack.com/archives/DT44VP17A/p1756839736829449)

> (regarding our easy-to-find, easy-to-understand Meetup docs and service :mild_panic: )
> tl;dr - Meetup is pissing me off!  They're absolute shysters.  We are not able to get emails.
> Today is the last day of the trial, and I thought it would be easy-peasy to get email addresses.  But it turns out email info is only available to organizers for events created after upgrading to pro.  So unless we've paid for a pro subscription since CFA Network dissolved in spring 2023, we would never have been able to get the emails anyway.  (And of course, this info is hidden on some [2022 blog post](https://www.meetup.com/blog/recording-unlocking-the-power-of-meetup-pro/), not any actual docs.)
> Following your [message](https://open-austin.slack.com/archives/DT44VP17A/p1756155213837749?thread_ts=1756155030.910269&cid=DT44VP17A), here's my plan to close this out properly.
> 1. ~~getting the emails out of the system~~ not possible
> 2. Every two weeks until the end of October, I will direct meetup group participants to IG, LinkedIn, and our [newsletter signup](https://docs.google.com/forms/d/e/1FAIpQLSfh8tClMhUQOcXzFXUq2H1yNElpzFExOdRMzJ1txTc-vn0lbQ/viewform).  Contact will be thru email and the message board (pictures)
> 3. At the upcoming practice meets, we will tell people about ^^^
> 4. closing down the account properly so it can't be used by anyone else.
> Anything I missed in the points above?

Aslan French [3:15 PM](https://open-austin.slack.com/archives/DT44VP17A/p1756844134745169)
Ah dang. Well I guess that's the situation.
I will defer to you on the event platform question. Your plans for getting the word out about a migration makes sense to me.
